### PR TITLE
Improved scripture reading experience

### LINKED ISSUE #265: Scripture Styling Tweaks
## User Story

As an app user I want to be focused on what matters so that I connect with God and grow within my church community.

#### Problem Details

There are a lot of different possible classes that could potentially need to be styled for scripture to display correctly. So far we've only done those that we've seen while testing select passages. The API.Bible folks have put together a whole bunch of styles for all the possible types [here](https://github.com/americanbible/api-bible-assets/tree/master/scss/eb-scripture-style/modules). We should manually add as many of these styles as necessary and style them correctly.

### PR DESCRIPTION
**Pull request by burkeshartsis**
Fixes #265 

## Creator

- [x] Build relevant tests if any apply
- [x] Ensure there are no new warnings (in app AND in storybook)
- [x] Upload an ~~animated GIF~~ image showcasing the solution (if it applies)
- [x] Set a relevant reviewer
- [x] PR has a relevant title that adheres to our patch-notes/change-log style

## Reviewer

- [ ] Review CI build results
- [ ] Review function and form on iOS
- [ ] Review function and form on Android
- [ ] Review new stories if any apply
- [ ] Review test coverage: Are all new or changed states tested?
- [ ] Review new test logic
- [ ] Review that the PR title adheres to our patch-notes/change-log style

### CODE REVIEW

**Comment by burkeshartsis** [comment]
@conrad-vanl this is ready for you. If this PR taught me anything it's that if we want to have more control over how we space things we are going to need to change how we parse our bible source to give us more control over the structure so we can style things better.

File: packages/apolloschurchapp/src/ui/Scripture/ScriptureHTMLView.js [hunk 1]
```diff
@@ -1,66 +1,135 @@
...
-import { Text } from 'react-native';
+import { Text, Platform } from 'react-native';
 
 import HTMLView from 'apolloschurchapp/src/ui/HTMLView';
 import defaultRenderer, {
   wrapTextChildren,
 } from 'apolloschurchapp/src/ui/HTMLView/defaultRenderer';
 import Paragraph from 'apolloschurchapp/src/ui/typography/Paragraph';
-import { H4, H6, BodyText } from 'apolloschurchapp/src/ui/typography';
+import { H4, SerifText } from 'apolloschurchapp/src/ui/typography';
```
Comment (conrad-vanl) [inline packages/apolloschurchapp/src/ui/Scripture/ScriptureHTMLView.js]: Thoughts on decision to name this `SerifText`? What if the specific church decides to use a serif font for `BodyText`? Or conversely, they want a "modern" typeface for scripture?

Comment (burkeshartsis) [reply packages/apolloschurchapp/src/ui/Scripture/ScriptureHTMLView.js]: Great point... I'm not sure I have a good answer. `PrimaryText` and `SecondaryText`? `ThemedTextPrimary` and `ThemedTextSecondary`? `DefaultText` and `AlternateText`? Even if we came up with an abstract enough naming convention for the components we would have issues matching the theme definitions. For example, theme is really only setup to handle 1 custom sans-serif and serif typeface respectively with the current naming. If someone wanted 2 custom sans-serif typefaces they would have to come up with their own naming, duplicate the needed type components, and adjust them appropriately. We could release a couple different common font stack packages that does this for them.

- Sans-Serif
- Serif
- Sans-Serif + Sans-Serif
- Serif + Serif
- Sans-Serif + Serif
- Sans-Serif + Sans-Serif + Sans-Serif
- Sans-Serif + Sans-Serif + Serif
- Sans-Serif + Serif + Serif
- Serif + Serif + Serif

😵

Comment (conrad-vanl) [reply packages/apolloschurchapp/src/ui/Scripture/ScriptureHTMLView.js]: Yeah, I hear yuh. `ScriptureText` ? Some component naming that says: "THIS <TEXT/> IS USED FOR THE WORD OF GOD"

Comment (burkeshartsis) [reply packages/apolloschurchapp/src/ui/Scripture/ScriptureHTMLView.js]: `ScriptureText` sounds perfect!

Comment (burkeshartsis) [reply packages/apolloschurchapp/src/ui/Scripture/ScriptureHTMLView.js]: Updated!

**Pull request merged**